data/trailhead.ftl

```diff
@@ -1,66 +1,134 @@
-onboarding-welcome-body = You’ve got the browser. Let’s explore the rest of Firefox.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-onboarding-membership-form-header = Set Up Your Firefox Membership
-onboarding-membership-form-email =
-    .placeholder = Your email …
-onboarding-membership-form-legal-links = By proceeding, you agree to the <a data-l10n-name="terms">Terms of Service</a> and <a data-l10n-name="privacy">Privacy Notice</a>.
-onboarding-membership-form-continue = Continue
+## The following feature names must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
 
+-facebook-container-brand-name = Facebook Container
+-lockwise-brand-name = Firefox Lockwise
+-monitor-brand-name = Firefox Monitor
+-pocket-brand-name = Pocket
+-send-brand-name = Firefox Send
 
-## These are individual items shown with an image, title and description.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-onboarding-subhead-productive-title = Be productive
-onboarding-subhead-productive-text = Get your everyday done with the browser, features, and apps.
+## UI strings for the simplified onboarding modal
 
-onboarding-subhead-smarter-title = Be a smarter browser
-onboarding-subhead-smarter-text = Get doable advice from world-class experts.
+onboarding-button-label-learn-more = Learn More
```

Comment:
I was thinking of removing the ones that already exist, as modifying the trailhead.ftl won't do anything as onboarding.ftl versions will be preferred... but then nahh I'll just plain concat the two brandings.ftl and onboarding.ftl files together 😉